integration/network/service_test.go

```diff
@@ -36,9 +36,67 @@ func TestServiceWithPredefinedNetwork(t *testing.T) {
 		if runtime.GOARCH == "arm64" || runtime.GOARCH == "arm" {
 			config.Timeout = 50 * time.Second
 			config.Delay = 100 * time.Millisecond
+		} else {
+			config.Timeout = 30 * time.Second
+			config.Delay = 100 * time.Millisecond
+		}
+	}
+
+	serviceID := serviceResp.ID
+	poll.WaitOn(t, serviceRunningCount(client, serviceID, instances), pollSettings)
+
+	_, _, err = client.ServiceInspectWithRaw(context.Background(), serviceID, types.ServiceInspectOptions{})
+	require.NoError(t, err)
+
+	err = client.ServiceRemove(context.Background(), serviceID)
+	require.NoError(t, err)
+
+	poll.WaitOn(t, serviceIsRemoved(client, serviceID), pollSettings)
+	poll.WaitOn(t, noTasks(client), pollSettings)
+
+}
+
+const ingressNet = "ingress"
```

Comment:
Do we also need a test with a custom ingress network?

Reply:
That's a fair question.  I'm not aware of any code that tags any other networks besides a swarm-mode network named "ingress" as an ingress network.  Neither am I aware of our support for such a construct.  However, that may just be my lack of experience w/ the product.

My use of the named constant was more out of a general purpose aversion to magic constants sprinkled through code.  (i.e. collecting them to a named token makes refactoring easier and saner as a rule ... but will probably not be needed in this case)

Reply:
Regarding "any code that tags any network besides swarm-mode network named "ingress" as an ingress network": you can create an overlay network named anything after removing existing ingress as long as it marked with `--ingress` flag. E.g. (after deleting existing ingress) `docker network create --ingress -d overlay testingress` works